ElevenLabs-DotNet/TextToSpeech/TextToSpeechEndpoint.cs

```diff
@@ -87,11 +87,13 @@ public async Task<VoiceClip> TextToSpeechAsync(string text, Voice voice, VoiceSe
             {
                 parameters.Add(OptimizeStreamingLatencyParameter, optimizeStreamingLatency.ToString());
             }
-            var postRequest = new HttpRequestMessage(HttpMethod.Post, GetUrl($"/{voice.Id}{(partialClipCallback == null ? string.Empty : "/stream")}", parameters))
-            {
-                Content = payload,
-            };
-            var response = await Api.Client.SendAsync(postRequest, partialClipCallback == null ? HttpCompletionOption.ResponseContentRead :  HttpCompletionOption.ResponseHeadersRead, cancellationToken);
+
+            using var postRequest = new HttpRequestMessage(HttpMethod.Post, GetUrl($"/{voice.Id}{(partialClipCallback == null ? string.Empty : "/stream")}", parameters));
```

Comment:
why the using here? without a scope?

Reply:
`HttpRequestMessage` implements `IDisposable`